content/applications/inventory_and_mrp/quality/quality_management/failure_locations.rst

```diff
@@ -0,0 +1,107 @@
+=================
+
+.. |QCP| replace:: :abbr:`QCP (Quality Control Point)`
+.. |QCPs| replace:: :abbr:`QCP (Quality Control Points)`
+
+In Odoo, quality control points (QCPs) are used to create quality checks, which prompt employees to
+confirm the quality of products, when they are included in certain operations. By setting one or
+more *failure locations* on a |QCP|, products that fail the quality checks it creates can be sent to
+one of the specified locations.
+
+.. important::
+   The *Failure Location* feature was added in version 17.0 of Odoo, and does not appear in any
+   previous version. To upgrade an Odoo database to a more recent version, see the documentation on
+   :doc:`database upgrades <../../../../administration/upgrade>`.
+
+Configuration
+=============
+
+To use failure locations, the *Storage Locations* setting must be enabled in the settings of the
+*Inventory* app. This setting allows for the creation of sub-locations within a warehouse, including
+failure locations.
+
+To enable the Storage Locations setting, navigate to :menuselection:`Inventory app --> Configuration
+--> Settings`, and tick the checkbox next to :guilabel:`Storage Locations`, under the
+:guilabel:`Warehouses` heading.
+
+.. image:: failure_locations/storage-locations-setting.png
+   :align: center
+   :alt: The Storage Locations setting on the Inventory app settings page.
+
+.. important::
+   Failure locations are most effective when used for products configured as *storable products*.
+   This is because inventory counts are only tracked for storable products, versus *consumable*
+   products, for which exact counts are not tracked.
+
+   Quality checks can still be created for consumable products, and those products can be sent to a
+   failure location if they fail a check. However, Odoo does not track the exact quantity of a
+   consumable product stored at a failure location.
+
+   To configure a product as storable, navigate to :menuselection:`Inventory app --> Products -->
+   Products`, and select a product. In the :guilabel:`Product Type` field on the :guilabel:`General
+   Information` tab, make sure that :guilabel:`Storable Product` is selected from the drop-down
+   menu.
+
+Add failure location to QCP
+===========================
+
+To add a failure location to a |QCP|, navigate to :menuselection:`Quality app --> Quality Control
+--> Control Points`. Select an existing |QCP| from the list, or create a new one by clicking
+:guilabel:`New`.
+
+.. note::
+   The following instructions only detail the configuration settings necessary for adding a failure
+   location to a |QCP|. For a full overview of |QCPs| and all of the options available when
+   configuring them, see the documentation on :doc:`quality control points
+   <quality_control_points>`.
+
+In the :guilabel:`Control Per` field on the |QCP| form, select the :guilabel:`Quantity` option.
+Doing so causes a :guilabel:`Failure Locations` field to appear on the form. This field is only
```

Comment:
```suggestion
Doing so causes a :guilabel:`Failure Locations` field to appear on the form. This field is **only**
```
Correct me if I'm wrong, doesn't the 'Partial Test' field *also* appear when the 'Quantity' option is selected? If so, do you think that might be worth mentioning here?

Reply:
I've included a note above stating that this doc only includes information relevant to failure locations. Details about the rest of the settings can be found in the link to the QCP doc.